qos_capacity.adoc

```diff
@@ -7,40 +7,40 @@ interface.
 
 The capacity controller allocates capacity in fixed multiples of _capacity
 units_. A group of these _capacity units_ is referred to as a _capacity block_.
-One or more _capacity blocks_ may be allocated to a workload. When a workload
-requests capacity allocation, the capacity is allocated using _capacity units_
+One or more _capacity blocks_ can be allocated to a workload. When a workload
+requests capacity allocation, the capacity is allocated by using _capacity units_
 situated within the _capacity blocks_ assigned to the workload. Capacity blocks
 can also be shared among one or more workloads. Optionally, the capacity
-controller may allow configuration of a limit on the maximum number of _capacity
-units_ that can be occupied in the _capacity blocks_ allocated to a given
+controller might allow configuration of a limit on the maximum number of _capacity
+units_ that can be occupied in the _capacity blocks_ allocated to a specific
 workload.
 
 [NOTE]
 ====
-For example, a cache controller may allocate capacity in multiples of cache
+For example, a cache controller allocates capacity in multiples of cache
 blocks. In this context, a cache block serves as a _capacity unit_, and a group
 of cache blocks forms a _capacity block_. A cache controller supporting capacity
-allocation by ways might define a _capacity block_ to be the cache blocks in one
+allocation might define a _capacity block_ to be the cache blocks in one
```

Comment:
@kersten1 - "by ways" is  important. In a set-associative cache is organized as a bunch (e.g., 512) of "sets" with each "set" associated with a bunch of "ways" (e.g., 8). In such a cache the allocations may be " by ways" i.e. in increments of ways. For the example with 8 ways the number  of _capacity blocks_ is then 8 i.e. the number of "ways" and the capacity represented  by that block is the sum of all cache blocks in that way across all sets (in this example, each "way" has 512  cache blocks)

Reply:
is this specifically a QoS thing?  Can we add some of your description in? I was totally baffled and when I searched the priv and unpriv, I didn't find "by ways" anywhere. So then I tried the ARM QoS docs, but didn't really find anything there either. So I think it would be good to spell it out a bit more here.

Reply:
Its a way of designing caches which priv and unpriv specs do not delve into. 
Please see https://www.sciencedirect.com/topics/computer-science/set-associative-cache